cognite/pygen/_query/executor.py

```diff
@@ -36,7 +34,7 @@ class Progress:
 
     def _update_nodes_per_second(self, last_node_count: int, last_execution_time: float) -> None:
         # Estimate the number of nodes per second using exponential moving average
-        last_batch_nodes_per_second = last_node_count / last_execution_time
+        last_batch_nodes_per_second = last_node_count / max(last_execution_time, 1e-6)
```

Comment:
This bug can only happen if you run in a test when execution time becomes 0.0. Thus not mentioned in the changelog.